Lint检查失败: builtin-commands.ts 存在 require-await 和 prefer-destructuring 错误

## 测试输出错误信息

\`\`\`
/Users/hs3180/clawd/disclaude/src/nodes/commands/builtin-commands.ts
  567:3   error  Async method 'execute' has no 'await' expression  require-await
  614:11  error  Use array destructuring                           prefer-destructuring

✖ 60 problems (2 errors, 58 warnings)
\`\`\`

## 根本原因分析

新提交的代码 \`src/nodes/commands/builtin-commands.ts\` 中存在两个 ESLint 错误：

1. **Line 567**: \`execute\` 方法被声明为 async 但没有使用 await 表达式
   - 需要移除 async 关键字或添加 await 调用

2. **Line 614**: 应使用数组解构赋值
   - 违反了 prefer-destructuring 规则

## 触发提交

- 08ff326 fix(mcp): add graceful degradation for Feishu tools without credentials (#579)
- ad85816 fix(lint): resolve ESLint errors in messaging module (Issue #572) (#574)
- 98d691a feat(schedule): add schedule control commands (Issue #469) (#573)

## 建议修复

1. 检查 line 567 的 async 方法，如果不需要 await 则移除 async 关键字
2. 检查 line 614 的赋值语句，改用数组解构 (如 \`const [a, b] = arr\` 而非 \`const a = arr[0]\`)

---
🤖 由 CI Monitor 自动创建

--- a/src/nodes/commands/builtin-commands.ts
+++ b/src/nodes/commands/builtin-commands.ts
@@ -607,11 +607,11 @@ export class ScheduleCommand implements Command {
 
     // Handle list subcommand
     if (subCommand === 'list') {
-      return this.handleList(services);
+      return await this.handleList(services);
     }
 
     // Other subcommands require a task name
-    const taskName = context.args[1];
+    const [, taskName] = context.args;
     if (!taskName) {
       return {
         success: false,
@@ -622,13 +622,13 @@ export class ScheduleCommand implements Command {
     // Handle other subcommands
     switch (subCommand) {
       case 'status':
-        return this.handleStatus(services, taskName);
+        return await this.handleStatus(services, taskName);
       case 'enable':
-        return this.handleEnable(services, taskName);
+        return await this.handleEnable(services, taskName);
       case 'disable':
-        return this.handleDisable(services, taskName);
+        return await this.handleDisable(services, taskName);
       case 'run':
-        return this.handleRun(services, taskName);
+        return await this.handleRun(services, taskName);
       default:
         return { success: false, error: `未知子命令: ${subCommand}` };
     }

